src/commands/kv/bucket/delete.rs

```diff
@@ -0,0 +1,22 @@
+use std::fs::metadata;
+use std::path::Path;
+
+use crate::commands::kv::bucket::directory_keys_only;
+use crate::commands::kv::bulk::delete::delete_bulk;
+use crate::settings::global_user::GlobalUser;
+use crate::settings::target::Target;
+
+pub fn delete(
+    target: &Target,
+    user: GlobalUser,
+    namespace_id: &str,
+    filename: &Path,
+) -> Result<(), failure::Error> {
+    let keys: Result<Vec<String>, failure::Error> = match &metadata(filename) {
+        Ok(file_type) if file_type.is_dir() => directory_keys_only(filename),
+        Ok(_) => failure::bail!("{} should be a directory", filename.display()),
+        Err(e) => failure::bail!("{}", e),
+    };
+
+    delete_bulk(target, user, namespace_id, keys?)
```

Comment:
does this command now need to give output?

Reply:
It does!

Reply:
this is handled in #601